BioCrypt-Personal-V0.4 Source-code/main.py

```
@@ -0,0 +1,1814 @@
+        restored_file_path = os.path.join(
+            restore_folder, os.path.basename(file_path).replace(".enc", ""))
+
+        with open(restored_file_path, 'wb') as f:
+            f.write(original_data)
+
+        return restored_file_path
+
+    # Function to generate and send OTP via email
+
+    def send_otp(email):
+        otp = str(random.randint(100000, 999999))  # Generate OTP
```

Ruff (0.12.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)